torchci/clickhouse_queries/queued_jobs_by_label/query.sql

```diff
@@ -47,3 +45,4 @@ GROUP BY
     machine_type
 ORDER BY
     count DESC
```

Comment:
Don't you need a semicolon here? Or is `SETTINGS` a keyword to `SELECT`?)
```suggestion
    count DESC;
```

Reply:
`SETTINGS` is a part of `SELECT`
https://clickhouse.com/docs/en/sql-reference/statements/select/from#final-modifier